### PR TITLE
Add hdp3.1 image (v2)

### PR DESCRIPTION
**Pull request by findepi**
Supersedes https://github.com/prestosql/docker-images/pull/13

### CODE REVIEW

**Comment by findepi** [comment]
(At this stage this is exact copy of https://github.com/prestosql/docker-images/pull/13)

File: etc/compose/hdp3.1-hive/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+version: '2.0'
```
Comment (kokosing) [inline etc/compose/hdp3.1-hive/docker-compose.yml]: side note: I don't know why we tend to copy conventions without much thinking ;) I think these files should be next to Dockerfiles. There is no point for having `etc/compose/..`. Also in presto codebase it is also overly complicated, but I am afraid of doing any changes there (to not to complicate things further).

Comment (findepi) [reply etc/compose/hdp3.1-hive/docker-compose.yml]: > I don't know why we tend to copy conventions without much thinking

Because thinking is expensive 😉

Comment (kokosing) [reply etc/compose/hdp3.1-hive/docker-compose.yml]: https://github.com/prestosql/docker-images/issues/15

**Pull request merged**